### PR TITLE
Fixes throwing 404

### PR DESCRIPTION
**Pull request by palmiak**
## Changes
When using the docs example, after running `npm run build` some CSS were missing:
image

After changing the imports everything started working again.

## Testing

I tested this on localhost and on cloudflare pages. Both with `npm start` and `npm run build`/

## Docs

It doesn't requires any update in my opinion.

### CODE REVIEW

File: examples/docs/public/index.css [hunk 1]
```diff
@@ -1,5 +1,5 @@
-@import './theme';
-@import './code';
+@import './../theme.css';
```
Comment (jasikpark) [inline examples/docs/public/index.css:1]: `./../theme.css` is the same as `../theme.css` btw, you might rewrite it as that.. this looks like a bug imo in general though..

Comment (matthewp) [reply examples/docs/public/index.css:1]: @palmiak can you update this?

**Comment by jasikpark** [comment]
There seems to be a bug w/ CSS bundling? in examples/docs index.css imports "./theme.css" and "./code.css" at the top of the file, then index.css gets pulled out into /_astro/common-2uStr8.css without rewriting the @import rules. 

This PR will fix the immediate problem, but this reads as a bug to me, unless if `public/` is meant to transform items in it?

**Comment by palmiak** [comment]
I also thought that import should import those files into one file (like it
happens in scss). We could transform it into scss and it would also solve
the problem

czw., 29 lip 2021, 23:51 użytkownik Caleb Jasik ***@***.***>
napisał:

**Comment by FredKSchott** [comment]
We actually import all of these files individually inside of `HeadCommon`, so there is no need to `@import` them as well. I updated your PR to just remove the two lines, and the preview URL looks good.

**Pull request merged**